flow/designs/asap7/mock-array/README.md

```diff
@@ -4,20 +4,22 @@ Mock array big
 A mock array that can be configured in different sizes to generate
 interesting test-cases.
 
-By default, the array is 8x8 elements and has an 8 bit datapath.
+By default, the array is 8x8 elements, has an 8 bit datapath with 
+Element size of 4x4 unit and Element placement pitch of 6,6 - x,y direction.
```

Comment:
change example to use unique values for each of the positions to clarify... Such as 1 2 3 4 5 6

Reply:
Will do